web-dev-server.config.js

```diff
@@ -1,10 +1,12 @@
 import { hmrPlugin, presets } from '@open-wc/dev-server-hmr';
 import json from '@rollup/plugin-json';
+import { nodeResolve } from '@rollup/plugin-node-resolve';
 import { fromRollup, rollupAdapter } from '@web/dev-server-rollup';
 import { storybookWdsPlugin } from './src/stories/lib/markdown.cjs';
 import { cemAnalyzerPlugin } from './wds/cem-analyzer-plugin.js';
 import { commonjsPluginWithConfig, esbuildBundlePluginWithConfig } from './wds/wds-common.js';
 
+console.log('CONFIG HERE');
```

Comment:
nitpick: maybe we can remove this console.log

Reply:
oupsie yes, thanks @Galimede